core/core-workflow-steps/src/main/java/org/eclipse/sw360/antenna/csvreader/CSVReader.java

```diff
@@ -395,10 +399,16 @@ private static String mapCPEIdToString(Artifact artifact) {
     }
 
     private static String mapFileNameToString(Artifact artifact) {
```

Comment:
I think the naming here should be adapted, actually it is more something like getFilepathAsString(), right?